Tests/Clappr_Tests/PlayButtonTests.swift

```diff
@@ -0,0 +1,234 @@
+import Quick
+import Nimble
+
+@testable import Clappr
+
+class PlayButtonTests: QuickSpec {
+    override func spec() {
+        describe(".PlayButton") {
+
+            describe("#init") {
+                it("is an MediaControlPlugin type") {
```

Comment:
Maybe we can have one instance of PlayButton to do all these 4 tests:
- `init`
- `pluginName `
- `panel `
- `position`

Reply:
Done.